### DISCUSSION TITLE
Any solutions for system alert on Android with device.openURL ?

### DISCUSSION THREAD

**Discussion by lukebrandonfarrell** [wix/Detox#2937] [Q&A] [answered]
When we use `device.openURL` it gives us the following (which breaks testing):

![Screenshot_1628606708](https://user-images.githubusercontent.com/18139277/128888196-9d07d45d-d9ec-4dd2-8937-55a73814af3f.png)

Are there any solutions to this?

**Comment by d4vidi** [comment, marked as answer]
@lukebrandonfarrell I'm going to fwd the discussion to #2504, and it's associated post on S/O - so as to avoid dups.

**Reply by lukebrandonfarrell** [reply]
Sure! Thanks @d4vidi
